### PR TITLE
Refactor variable and parameter names.

### PR DESCRIPTION
**Pull request by homotechsual**

### CODE REVIEW

File: Public/Get-HuduArticles.ps1 [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+		$Article = Invoke-HuduRequest -Method get -Resource "/api/v1/articles/$Id"
 		return $Article
 	} else {
 
 		$resourcefilter = ''
 
-		if ($companyid) {
-			$resourcefilter = "$($resourcefilter)&companyid=$($companyid)"
+		if ($CompanyId) {
+			$resourcefilter = "$($resourcefilter)&companyid=$($CompanyId)"
```
Comment (homotechsual) [inline Public/Get-HuduArticles.ps1:18]:
```suggestion
			$resourcefilter = "$($resourcefilter)&company_id=$($CompanyId)"
```

@lwhitelock is this change needed?

Comment (lwhitelock) [reply Public/Get-HuduArticles.ps1:18]: Line 6 $Iame needs to be $Name
